app/models/ems_refresh/save_inventory_container.rb

```diff
@@ -395,6 +396,19 @@ def save_container_volumes_inventory(container_group, hashes, target = nil)
     store_ids_for_new_records(container_group.container_volumes, hashes, :name)
   end
 
+  def save_additional_attributes_inventory(entity, hashes, target = nil)
```

Comment:
> advanced_settings has default_value, min, max, and read_only attributes.

that's because it was built from vmware which gave us those values.  advanced_settings there were modifiable, but not common to modify and definitely not modified by us.  custom_attributes was closer to like amazon tags or openshift labels, where it was quite common to change/add/delete them.  So if this is what these hawkular settings are, then it belongs in custom_attributes.